### DISCUSSION TITLE
Angles between constellation lines

### DISCUSSION THREAD

**Discussion by GeorgeIonStellarium** [Stellarium/stellarium#2252] [Q&A]
I am currently working on a project trying to determine the angles between the three stars of the Orion's belt through time, trying to figure if there is a closer angle to a hypothetical similar angle drawn by the pyramids as stellar maps marks thousands of years in past (trying to figure the time of construction of the pyramids).

So far i tried with the angle measuring tool and time window. Current angle measuring tool is hard to use as is does not snap on stars and especially cannot measure angle between two lines, but only relative to horizon or grid and positioning of the ends is difficult and uncertain and it does not stay on stars as i change time. Also, after coming back in the same time as i try to measure an angle again, angle seem to be constantly changing through time rolling is paused.

**Comment by gzotti** [comment]
Please read Appendix F.2 in the User Guide before you decide to use Stellarium for tasks like this.

**Reply by GeorgeIonStellarium** [reply]
Can't read Appendix F.2 or any other cause i'm illiterate.
